components/x-eventpromo/src/Image.jsx

```diff
@@ -0,0 +1,16 @@
+import { h } from '@financial-times/x-engine';
+import Link from './Link';
+
+const Image = ({linkUrl, imageUrl}) => {
+
+	const baseSrc = `https://www.ft.com/__origami/service/image/v2/images/raw/${imageUrl}?source=next&amp;fit=cover&amp;compression=best&amp;`;
+	const imgSrc = `${baseSrc}width=340 340w,${baseSrc}width=400 740w`;
+
+	return (
+		<Link url={linkUrl} attrs={{ 'className': 'js-event-promo--animate', 'data-trackable': 'event-promo'}}>
+			<img className="js-event-promo--animate event-promo-inarticle__img fade-0" srcSet={imgSrc} alt="" data-n-image-lazy-load-js="" role="presentation" styles={{size: '(max-width: 46.24em) 100vw, 45vw'}} />
```

Comment:
Not sure what's going on with the `styles` property here, do you mean `sizes`?

Reply:
Because the image has no content for an assistive device to read it may be best to hide it from them by adding `aria-hidden="true"` and `tabindex="-1"` to the link